src/main/fc/config.c

```diff
@@ -594,19 +607,13 @@ void readEEPROM(void)
     if (!loadEEPROM()) {
         failureMode(FAILURE_INVALID_EEPROM_CONTENTS);
     }
+
+    validateAndFixConfig();
+    validateAndFixGyroConfig();
```

Comment:
I think this has to be guarded by `#ifndef USE_OSD_SLAVE`.

Reply:
Yes, thanks.